.github/workflows/ci-cd.yml

```diff
@@ -35,16 +35,16 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - name: Check out repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

Comment:
@coderabbitai can you verify that all SHA in all changed actions match the most recent versions of those actions?